source/common/router/upstream_request.cc

```diff
@@ -517,126 +517,5 @@ void UpstreamRequest::enableDataFromDownstreamForFlowControl() {
   }
 }
 
-void HttpConnPool::newStream(GenericConnectionPoolCallbacks* callbacks) {
```

Comment:
I'm not sure what the policy is on code vs config location for "core" code.
Something like the raw buffer socket has config in extensions and source in core but for tls we have both code and config in extensions.  I assume code and config colocated is the way to go but lmk if you disagree

Reply:
Yeah agreed and yeah we have not been consistent here, probably most for historical code movement reasons.